modules/bbp-core/src/main/resources/schemas/bbp/core/mediatype/v0.1.0.json

```diff
@@ -0,0 +1,159 @@
+{
+  "@context": [
+    "{{base}}/contexts/nexus/core/schema/v0.1.0",
+    {
+      "ianaform": "https://www.iana.org/form/media-types/",
+      "this": "{{base}}/schemas/bbp/core/mediatype/v0.1.0/shapes/"
+    }
+  ],
+  "@type": "nxv:Schema",
+  "imports": [
+    "{{base}}/schemas/nexus/core/mediatype/v0.1.0",
+    "{{base}}/schemas/nexus/schemaorg/propertyvalue/v0.1.0"
+  ],
+  "shapes": [
+    {
+      "@id": "this:MediaTypeShape",
+      "@type": "sh:NodeShape",
+      "label": "A shape for describing a media type.",
+      "comment": "This shape is inspired from the IANA media type registration form. Please review existing types and subtypes of  media type (see https://www.iana.org/assignments/media-types/media-types.xhtml ). ",
```

Comment:
Remove the extra space after [...].xhtml

Reply:
done